app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -95,7 +95,7 @@ public void signUp() {
         if(personalServer.isChecked()) {
             if(!CredentialHelper.checkIfEmpty(url,this) && CredentialHelper.isURLValid(url,this)) {
                 if (CredentialHelper.getValidURL(url,this) != null) {
-                    PrefManager.putBoolean("is_susi_server_selected", false);
+                    PrefManager.putBoolean(String.valueOf(R.string.action_select_server), false);
```

Comment:
This is not the correct way of doing it. This will convert the autogenerated string id integer to a string and this is being used for saving preferences. This can change on each build itself. Please use `context.getString`